packages/editor/src/components/mobile/bottom-sheet/styles.scss

```diff
@@ -17,7 +17,6 @@
 	background-color: $light-gray-400;
 	height: 1px;
 	width: 100%;
-	margin-bottom: 14px;
```

Comment:
This file should contain `.native.scss` extension given that is used only in the mobile app context.